clang/test/Sema/constexpr.c

```diff
@@ -391,3 +391,10 @@ void ghissue109095() {
   _Static_assert(i == c[0]); // expected-error {{static assertion expression is not an integral constant expression}}\
                              // expected-note {{initializer of 'i' is not a constant expression}}
 }
+
+typedef bool __vbool2  __attribute__((ext_vector_type(2)));
+typedef short v2int16_t __attribute__((ext_vector_type(2)));
+
+bool issue155507(v2int16_t a, v2int16_t b) {
```

Comment:
This test file was intended for exercising c23 constexpr specifier. The new case doesn't seem to be using it, maybe there is another test that is better for the new case?

Reply:
ok, no problem